app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,11 @@
 class PostsController < ApplicationController
   def index
+    @user = User.find_by_id(params[:user_id])
     @posts = Post.all
   end
```

Comment:
- [ ] The Post _**index**_ page should contain all the `posts` for a **_single user_**. I would advise that after getting the single user, you should get all the **_posts for that user_** and not all the **_posts in the database_** as you have it. Kindly make the change. 👍 😎